tests/provider/dwd/radar/test_index.py

```diff
@@ -48,172 +53,157 @@ def test_radar_fileindex_composite_rv_reflectivity_bin(default_settings):
     ],
 )
 def test_radar_fileindex_radolan_reflectivity_bin(parameter, default_settings):
-
     file_index = create_fileindex_radar(parameter=parameter, settings=default_settings)
-
-    urls = file_index["filename"].tolist()
+    assert not file_index.is_empty()
+    urls = file_index.get_column("filename").to_list()
     assert all(
         PurePath(url).match(f"*/weather/radar/radolan/{parameter.value}/*---bin")
         for url in urls
         if not url.endswith(".bz2")
     )
 
 
+@pytest.mark.remote
 def test_radar_fileindex_sites_px_reflectivity_bin(default_settings):
-
     file_index = create_fileindex_radar(
         parameter=DwdRadarParameter.PX_REFLECTIVITY,
         site=DwdRadarSite.BOO,
         fmt=DwdRadarDataFormat.BINARY,
         settings=default_settings,
     )
-
-    urls = file_index["filename"].tolist()
+    assert not file_index.is_empty()
+    urls = file_index.get_column("filename").to_list()
     assert all(PurePath(url).match("*/weather/radar/sites/px/boo/*---bin") for url in urls)
 
 
+@pytest.mark.remote
 def test_radar_fileindex_sites_px_reflectivity_bufr(default_settings):
-
     file_index = create_fileindex_radar(
         parameter=DwdRadarParameter.PX_REFLECTIVITY,
         site=DwdRadarSite.BOO,
         fmt=DwdRadarDataFormat.BUFR,
         settings=default_settings,
     )
-
-    urls = file_index["filename"].tolist()
+    assert not file_index.is_empty()
+    urls = file_index.get_column("filename").to_list()
     assert all(PurePath(url).match("*/weather/radar/sites/px/boo/*---buf") for url in urls)
 
 
+@pytest.mark.remote
 def test_radar_fileindex_sites_px250_reflectivity_bufr(default_settings):
-
     file_index = create_fileindex_radar(
         parameter=DwdRadarParameter.PX250_REFLECTIVITY, site=DwdRadarSite.BOO, settings=default_settings
     )
-
-    urls = file_index["filename"].tolist()
+    assert not file_index.is_empty()
+    urls = file_index.get_column("filename").to_list()
     assert all("/weather/radar/sites/px250/boo" in url for url in urls)
 
 
-def test_radar_fileindex_sites_sweep_bufr(default_settings):
-
-    file_index = create_fileindex_radar(
-        parameter=DwdRadarParameter.SWEEP_VOL_VELOCITY_H,
-        site=DwdRadarSite.BOO,
-        fmt=DwdRadarDataFormat.BUFR,
-        settings=default_settings,
-    )
-
-    urls = file_index["filename"].tolist()
-    assert all(PurePath(url).match("*/weather/radar/sites/sweep_vol_v/boo/*--buf.bz2") for url in urls)
-
-
```

Comment:
Is it gone?

Reply:
yes, bufr is gone :'(

Reply:
should I take this into another PR?

Reply:
> bufr is gone

Does it mean there is no bufr data any longer at the relevant location? How did the test succeed before?